apps/gauzy/src/app/pages/employees/edit-employee/edit-employee-profile/edit-employee-settings/edit-employee-other-settings.component.scss

```
@@ -52,10 +52,15 @@
       border-radius: nb-theme(button-rectangle-border-radius);
       list-style: none;
       cursor: pointer;
-      box-shadow: 0px 1px 1px rgba(0, 0, 0, 0.15);
+      // `gauzy-shadow`, not a literal black drop shadow. The token IS
```

Stylelint (17.14.1)

[error] Unknown word `gauzy-shadow`, (CssSyntaxError)

(CssSyntaxError)